packages/assets-controllers/src/NftDetectionController.ts

```diff
@@ -154,6 +149,231 @@ export interface NftDetectionConfig extends BaseConfig {
+  Spam = 'Spam',
+  Warning = 'Warning',
+  Malicious = 'Malicious',
+}
+
+export type Blockaid = {
+  contract: string;
+  chainId: number;
+  result_type: BlockaidResultType;
+  malicious_score: string;
+  attack_types: object;
+};
+
+export type Market = {
+  floorAsk?: FloorAsk;
+  topBid?: TopBid;
+};
+
+export type TokenResponse = {
+  chainId: number;
+  contract: string;
+  tokenId: string;
+  /** @description Can be erc721, erc115, etc. */
+  kind?: string;
+  name?: string;
+  image?: string;
+  imageSmall?: string;
+  imageLarge?: string;
+  metadata?: Metadata;
+  description?: string;
+  /** @description Can be higher than one if erc1155. */
+  supply?: number;
+  remainingSupply?: number;
+  /** @description No rarity for collections over 100k */
+  rarityScore?: number;
+  rarity?: number;
+  /** @description No rarity rank for collections over 100k */
+  rarityRank?: number;
+  media?: string;
+  /** @default false */
+  isFlagged?: boolean;
+  /** @default false */
+  isSpam?: boolean;
+  /** @default false */
+  isNsfw?: boolean;
+  /** @default false */
+  metadataDisabled?: boolean;
+  lastFlagUpdate?: string;
+  lastFlagChange?: string;
+  collection?: Collection;
+  lastSale?: LastSale;
+  topBid?: TopBid;
+  /** @description The value of the last sale.Can be null. */
+  lastAppraisalValue?: number;
+  attributes?: Attributes[];
+};
+
+export type TopBid = {
+  id?: string;
+  price?: Price;
+  source?: {
+    id?: string;
+    domain?: string;
+    name?: string;
+    icon?: string;
+    url?: string;
+  };
+};
+
+export type LastSale = {
+  /** @description Deprecated. Use `saleId` instead. */
+  id?: string;
+  /** @description Unique identifier made from txn hash, price, etc. */
+  saleId?: string;
+  token?: {
+    contract?: string;
+    tokenId?: string;
+    name?: string;
+    image?: string;
+    collection?: {
+      id?: string;
+      name?: string;
+    };
+  };
+  orderSource?: string;
+  /**
+   * @description Can be `ask` or `bid`.
+   * @enum {string}
+   */
+  orderSide?: 'ask' | 'bid';
+  orderKind?: string;
+  orderId?: string;
+  from?: string;
+  to?: string;
+  amount?: string;
+  fillSource?: string;
+  block?: number;
+  txHash?: string;
+  logIndex?: number;
+  batchIndex?: number;
+  /** @description Time added on the blockchain */
+  timestamp?: number;
+  price?: Price;
+  washTradingScore?: number;
+  royaltyFeeBps?: number;
+  marketplaceFeeBps?: number;
+  paidFullRoyalty?: boolean;
+  feeBreakdown?: FeeBreakdown[];
+  isDeleted?: boolean;
+  /** @description Time when added to indexer */
+  createdAt?: string;
+  /** @description Time when updated in indexer */
+  updatedAt?: string;
+};
+
+export type FeeBreakdown = {
+  kind?: string;
+  bps?: number;
+  recipient?: string;
+  source?: string;
+  rawAmount?: string;
+};
+
+export type Attributes = {
+  /** @description Case sensitive */
+  key?: string;
+  /** @description Can be `string`, `number, `date, or `range`. */
```

Comment:
Why is this not reflected in the type?

Reply:
This is a sample of how attribute would look like, 
```
          {
            "key": "Background",
            "kind": "string",
            "value": "Pink",
            "tokenCount": 2845,
            "topBidValue": null,
            "createdAt": "2022-06-22T17:01:57.431Z"
          },
```
OR
```
          {
            "key": "Drip Score",
            "kind": "number",
            "value": "33",
            "tokenCount": 409,
            "onSaleCount": 22,
            "topBidValue": 0.02985,
            "createdAt": "2022-06-22T17:01:57.431Z"
          }
```
So the kind is always a string, i guess the description was a bit confusing.
I do not expect we will be using the "kind", not for now at least, we would mainly use the "key", "value", "tokenCount" and floorPrices if they exist.. i can keep it for now unless we think we want to remove anything that is not used today then have it added in the future when we need it 🙏